### DISCUSSION TITLE
Quote admonitions not working under certain circumstances

### DISCUSSION THREAD

**Discussion by mjf** [requarks/wiki#6983] [Error / Bug Report] [answered]
Version: `2.5.300`, `2.5.301`

This works:

```markdown
1. list item
2. second list item

   > with some quote I want to express danger
   {.is-danger}

   extra list item content is needed here otherwise quote admonitions does not work

3. third item
```

This does not create the expected admonition:

```markdown
1. list item
2. second list item

   notice that there is no other list item content after the quote

   > with some quote I want to express danger
   {.is-danger}

3. third item
```

I think I saw this behaviour in some other circumstances too.

**Comment by mjf** [comment]
I also realized that list item rendering may be broken completely (not supporting paragraphs within list item results in weird rendering):

This does not work as expected:

```markdown
1. first list item paragraph

   second list item paragraph

   third list item paragraph
```

and therefor this yields quite unexpected results (blue list item with red and green paragraphs inside is the expeted result)

```markdown
1. list item

   red paragraph
   {.red}

   green paragraph
   {.green}

   normal paragraph
{.blue}
```

I only guess that this may be somehow connected to some indenting and parsing issues?

**Comment by NGPixel** [comment, marked as answer]
The parser can't guess which element to apply. Use the decorate syntax to specify the element to apply the classes to.

See https://github.com/rstacruz/markdown-it-decorate#disambiguating for docs and examples.

**Reply by mjf** [reply]
Thank you for the explanation, it clarifies the core of the issue very well.

**Comment by mjf** [comment]
No need for further discussion, I think.
